### DISCUSSION TITLE
Static buttons and automatic requests for lookup callsign data from national callbook web-sites

### DISCUSSION THREAD

**Discussion by R1BLH** [magicbug/Cloudlog#1764] [Ideas / Feature Requests]
It would bee good to have possibility lookup callsign data from national callbook web-sites like it done for qrz.com and hamqth.com.
It would be good that list of usable source data for lookup be configurable (like bands).

Now after every update from git i add lines for static buttons and menu for lookup in the national callbook (qrz.ru) manually into:
application/views/search/search_result_ajax.php
application/views/view_log/qso.php
application/views/view_log/partial/log_ajax.php
application/views/qso/index.php
assets/js/sections/qso.js

Result looks like:

![2022-11-10_16-51-34](https://user-images.githubusercontent.com/16667173/201109566-d208e12a-6f17-45cc-ad1f-d13a6bc12122.png)
![2022-11-10_16-38-56](https://user-images.githubusercontent.com/16667173/201109570-d64df201-9004-4052-adf5-96075c63cddf.png)
![2022-11-10_16-38-09](https://user-images.githubusercontent.com/16667173/201109573-2c958c11-dcb9-418e-a14a-381de9c74ca2.png)

**Comment by magicbug** [comment]
Agreed nice to have but i feel these need to be user optional else it could get busy quickly

**Reply by R1BLH** [reply]
Yes! It would be optional!
